first/expo/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Farmington Valley FIRST Expo
+---
+
+<!--<h2>Farmington Valley FLL Jr.</h2>
+This section will be updated with more information about the Farmington Valley FLL Jr. programs soon.
+-->
```

Comment:
You don't really have to keep this stuff that's commented out if it's not relevant anymore.